qa/src/test/java/io/camunda/migrator/qa/SubprocessMigrationTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+package io.camunda.migrator.qa;
+
+import static io.camunda.migrator.qa.MigrationTestConstants.LEGACY_ID_VAR_NAME;
+import static io.camunda.process.test.api.CamundaAssert.assertThat;
+import static io.camunda.process.test.api.assertions.ElementSelectors.byId;
+import static io.camunda.process.test.api.assertions.ProcessInstanceSelectors.byProcessId;
+import static io.camunda.process.test.api.assertions.UserTaskSelectors.byTaskName;
+
+import java.util.stream.Stream;
+import org.camunda.bpm.engine.runtime.ProcessInstance;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.provider.Arguments;
+
+public class SubprocessMigrationTest extends RuntimeMigrationAbstractTest{
+
+  @Test
+  public void migrateCallActivityAndSubprocess() {
+    // given
+    deployProcessInC7AndC8("calledProcessInstance.bpmn");
+    deployProcessInC7AndC8("callActivity.bpmn");
```

Comment:
Nit
the other bpmns have names line `...Process.bpmn`, could rename these cor consistency. Eg `
```suggestion
    deployProcessInC7AndC8("callActivitySubprocess.bpmn");
    deployProcessInC7AndC8("callActivityProcess.bpmn");
```

Reply:
Updated file names to `calledActivitySubprocess` & `callActivityProcess`